### DISCUSSION TITLE
New setting to display dictionary entries on plain mouse hover instead of SHIFT+hover?

### DISCUSSION THREAD

**Discussion by ghost** [ezra-bible-app/ezra-bible-app#864] [Ideas]
I'm just getting started with Ezra and it is a wonderful tool.  Thanks to everyone for your contributions.  

I make heavy use of Strong's in my reading.  I was wondering if it may be possible to add a new setting to the gear icon to display dictionary entries simply by hovering over the verse text, instead of requiring the SHIFT key.  Then a simple click on the word would lock the highlighted dictionary entry into the sidebar.  

The reason for this request is just to make dictionary look-ups a one-handed operation instead of two-handed.

There might be some technical reason I'm not aware of that the SHIFT requirement was decided on.  If it is technically feasible to do this, this would make dictionary navigation a bit easier.  

Another possible option would be middle-click (mouse wheel click) to load the dictionary entry for a word.  This would also permit one-handed navigation.

Thank you.

**Comment by tobias-klein** [comment]
@0x199x 
Thanks for your sharing your thoughts on this! I see your point about one-handed navigation.
An alternative to enabling the mouseover all the time would be to enable it for the currently selected verse. So you would first have to click on the verse and then mouseover would be enabled for that verse. What do you think about that alternative? I guess the disadvantage with permanently enabled mouseover is that you may accidentally change the selected word to one from another verse.

**Reply by ghost** [reply]
Yes that option would be just fine, thank you for acknowledging this.  Any improvement that assists with one-handed operation would be welcome.  It is really just a matter of comfort for long reading sessions.. easier to only mess with the mouse. 

To prevent the issue you describe, I suggested in the original post to use a simple left click to "lock" the selected word into the sidebar (essentially the same thing that happens when you release SHIFT over a word, currently).  Hovering over different words would still temporarily change the definition in the sidebar, but as soon as you move your cursor away from a word, the "locked" definition that you last clicked on would still be in the sidebar.  Mouse hover = preview, click = selection.
